### PR TITLE
Arch Pros & Cons finish

### LINKED ISSUE #3: [Arch] Strengths and Weaknesses
- Installation difficulty.
  - Software availability.  (i.e. How hard is it to install non-FOSS Software?)
  - Is `Flatkpak` pre-installed?

### PR DESCRIPTION
**Pull request by sara-saye**
closes #3

### CODE REVIEW

File: Arch/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      After loading the ISO file for installation, the user can freely monitor and control various functional aspects, which is a fundamental distinguishing feature of Arch Linux.
+* **Flexible package management and integrity**
+      LINUX ARCH supports PACMAN management. This technology provides the ability to upgrade the entire system with a single pacman command - Syu. Loading of new versions of packages is performed by one click of pacman - S package-name.
+* **Arch Wiki**
+      your source for Arch Linux documentation on the web.
+* **Originality** 
+      Unlike other distributions such as Ubuntu, which is based on Debian, the arch Linux operating system was created from the ground up, independent of any Linux distribution.
+* **Flexibility**
+      Working with Arch Linux is very pleasant in addition to its complexities. it is a very flexible operating system that you are able to change it as you like, Also You are free to add the required features by choosing modules from the time of its installation.     
+# cons
```
Comment (thisisamna) [inline Arch/README.md]: Capitalize the C

File: Arch/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Pros
+* **Customization capability**
+      After loading the ISO file for installation, the user can freely monitor and control various functional aspects, which is a fundamental distinguishing feature of Arch Linux.
+* **Flexible package management and integrity**
+      LINUX ARCH supports PACMAN management. This technology provides the ability to upgrade the entire system with a single pacman command - Syu. Loading of new versions of packages is performed by one click of pacman - S package-name.
```
Comment (thisisamna) [inline Arch/README.md]: Change "LINUX ARCH" to "Arch Linux" for consistency

File: Arch/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+* **Arch Wiki**
+      your source for Arch Linux documentation on the web.
```
Comment (thisisamna) [inline Arch/README.md]: Most distros have a wiki, so this alone isn't a pro. The real pro for Arch is that its wiki is rich, detailed, and up-to-date, making it superior to any other distro's wiki . Maybe we can say something about that here!

File: Arch/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+* **Originality** 
+      Unlike other distributions such as Ubuntu, which is based on Debian, the arch Linux operating system was created from the ground up, independent of any Linux distribution.
+* **Flexibility**
+      Working with Arch Linux is very pleasant in addition to its complexities. it is a very flexible operating system that you are able to change it as you like, Also You are free to add the required features by choosing modules from the time of its installation.     
+# cons
+* **Difficult to install**
```
Comment (thisisamna) [inline Arch/README.md]: Maybe let's not make these bold, and add a bit of detail. For example: 
- Can be difficult to install and set up for the average user
- Not very usable out of the box, and requires a lot of configuration

**Review by thisisamna — CHANGES_REQUESTED** [review]
Nice work!

**Comment by sara-saye** [comment]
All changes have done!

**Review by thisisamna — APPROVED** [review]
Awesome, thank you! Merging.

**Pull request merged**